### PR TITLE
more vulnerability remediation

### PR DESCRIPTION
**Pull request by damianxu88**

### CODE REVIEW

**Review by schowsf — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -49,6 +49,7 @@ lazy val ws = (project in file("acdc-ws")).
       kineticpulse
     ),
     dependencyOverrides ++= Seq(
+      "com.google.guava" % "guava" % "32.0.0-jre",
```
Comment (realstraw) [inline build.sbt:52]: what's the guava dependency it suppose to have?

Comment (realstraw) [reply build.sbt:52]: they also generally does not do well with dependencies, we might want to test that too

Comment (damianxu88) [reply build.sbt:52]: @realstraw sure, let me test it

**Comment by mfleming99** [comment]
should also override `org.flywaydb:flyway-commandline`?

**Comment by damianxu88** [comment]
@mfleming99 flyway is not part the dependencies in the OSS repo

**Pull request merged**